### ISSUE TITLE
Implement --help and --version

### ISSUE THREAD

**Issue by felixSchl** [felixSchl/docopt.js#48]
The developer will be able to specify the option that will trigger the `--help` and `--version` convenience functions, with fallbacks to `--help` and `--version`.
- [x] Implement `--help`. If matched, print the docopt text. ([feature/help](https://github.com/felixSchl/neodoc/tree/feature/help))
- [x] Implement `--version`. If matched, print either the npm package version or a user-provided version string

**Comment by moll** [comment]
Not overly difficult to do now, either:

``` js
if (args["--version"]) return void process.stdout.write(VERSION_TEXT)
```

**Comment by felixSchl** [comment]
Available in version 0.10.0 - https://github.com/felixSchl/neodoc/releases/tag/v0.10.0
